Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -916,6 +918,54 @@ Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<String>>> InspectorCSSAgent
     return fontFamilyNames;
 }
 
+Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<Inspector::Protocol::CSS::FontFace>>> InspectorCSSAgent::getCustomFonts()
+{
+    auto customFonts = JSON::ArrayOf<Inspector::Protocol::CSS::FontFace>::create();
+    auto* domAgent = m_instrumentingAgents.persistentDOMAgent();
+
+    for (auto* document : domAgent->documents()) {
```

Comment:
im not entirely sure what benefit there is in grabbing fonts for _all_ documents, as i dont think fonts are usable in situations like across `<iframe>`

perhaps we should adjust this to be something that gets the lists of fonts given a node (or more ideally a document node)?  that way we can show only the list of usable fonts for the selected node in the Fonts details sidebar panel